idaes/models/unit_models/pressure_changer.py

```diff
@@ -329,12 +332,117 @@ def head(self, t):  # units are energy/mass
+        # sufficient for most unit model applications.
+        for t in model.flowsheet().time:
+            self.scale_variable_by_default(model.ratioP[t], overwrite=overwrite)
+
+        # scale efficiencies. Default scaling factor is 10 which should be sufficient for most unit model applications -
+        if hasattr(model, "efficiency_pump"):
+            for t in model.flowsheet().time:
+                self.scale_variable_by_default(
+                    model.efficiency_pump[t], overwrite=overwrite
+                )
+
+        if hasattr(model, "efficiency_isentropic"):
+            for t in model.flowsheet().time:
+                self.scale_variable_by_default(
+                    model.efficiency_isentropic[t], overwrite=overwrite
+                )
+        # --------------------------------------------------------------------------------------------------------------
+
+        # for unit model work variables, we can simply use the scaling for the work variable in the control volume -
+        if hasattr(model, "work_fluid"):
+            for t in model.flowsheet().time:
+                self.scale_variable_by_component(
+                    model.work_fluid[t], model.control_volume.work[t]
+                )
+
+        if hasattr(model, "work_isentropic"):
+            for t in model.flowsheet().time:
+                self.scale_variable_by_component(
+                    model.work_isentropic[t], model.control_volume.work[t]
+                )
+
+        if hasattr(model, "work_mechanical"):
+            for t in model.flowsheet().time:
+                self.scale_variable_by_component(
+                    model.work_mechanical[t], model.control_volume.work[t]
+                )
+        # ----------------------------------------------------------------------------------------------------------
+
+    def constraint_scaling_routine(
+        self, model, overwrite: bool = False, submodel_scalers: ComponentMap = None
+    ):
+        # Call scaling method for control volume sub-model
+        self.call_submodel_scaler_method(
+            submodel=model.control_volume,
+            method="constraint_scaling_routine",
+            submodel_scalers=submodel_scalers,
+            overwrite=overwrite,
+        )
+
+        # Call the scaling method for the duplicate properties method
+        if hasattr(model, "properties_isentropic"):
+            self.call_submodel_scaler_method(
+                submodel=model.properties_isentropic,
+                method="constraint_scaling_routine",
+                submodel_scalers=submodel_scalers,
+                overwrite=overwrite,
+            )
+
+        # scale remaining unit level constraints
+        for c in model.component_data_objects(Constraint, descend_into=False):
+            self.scale_constraint_by_nominal_value(
+                c,
+                overwrite=overwrite,
+            )
+
+
```

Comment:
Are you sure that this constraint scaling scheme is appropriate for every constraint in the model?